shell/platform/windows/flutter_window_winuwp.cc

```diff
@@ -0,0 +1,337 @@
+#ifdef USECOREWINDOW
+  return WindowsRenderTarget(window_);
+#else
+  compositor_ = winrt::Windows::UI::Composition::Compositor();
+  target_ = compositor_.CreateTargetForCurrentView();
+  visual_tree_root_ = compositor_.CreateContainerVisual();
+  target_.Root(visual_tree_root_);
+
+  cursor_visual_ = CreateCursorVisual();
+
+  render_target_ = compositor_.CreateSpriteVisual();
+  if (running_on_xbox_) {
+    render_target_.Offset(
+        {xbox_overscan_x_offset_, xbox_overscan_y_offset_, 1.0});
+  } else {
+    render_target_.Offset({1.0, 1.0, 1.0});
+    ApplyInverseDpiScalingTransform();
+  }
+  visual_tree_root_.Children().InsertAtBottom(render_target_);
+
+  auto bounds = GetBounds(current_display_info_, true);
+
+  render_target_.Size({bounds.width, bounds.height});
+  return WindowsRenderTarget(render_target_);
+#endif
+}
+
+void FlutterWindowWinUWP::ApplyInverseDpiScalingTransform() {
+  // Apply inverse transform to negate built in DPI scaling in order to render
+  // at native scale.
+  auto dpiScale = GetDpiScale();
+  render_target_.Scale({1 / dpiScale, 1 / dpiScale, 1 / dpiScale});
+}
+
+PhysicalWindowBounds FlutterWindowWinUWP::GetPhysicalWindowBounds() {
+  auto bounds = GetBounds(current_display_info_, true);
+  return {static_cast<size_t>(bounds.width),
+          static_cast<size_t>(bounds.height)};
+}
+
+void FlutterWindowWinUWP::UpdateFlutterCursor(const std::string& cursor_name) {
+  // TODO(clarkezone): add support for Flutter cursors:
+  // https://github.com/flutter/flutter/issues/70199
+}
+
+void FlutterWindowWinUWP::UpdateCursorRect(const Rect& rect) {
+  // TODO(cbracken): Implement IMM candidate window positioning.
+}
+
+void FlutterWindowWinUWP::OnWindowResized() {}
+
+float FlutterWindowWinUWP::GetDpiScale() {
+  auto disp = winrt::Windows::Graphics::Display::DisplayInformation::
+      GetForCurrentView();
+
+  return GetDpiScale(disp);
+}
+
+WindowBoundsWinUWP FlutterWindowWinUWP::GetBounds(
+    winrt::Windows::Graphics::Display::DisplayInformation const& disp,
+    bool physical) {
+  winrt::Windows::UI::ViewManagement::ApplicationView app_view =
+      winrt::Windows::UI::ViewManagement::ApplicationView::GetForCurrentView();
+  winrt::Windows::Foundation::Rect bounds = app_view.VisibleBounds();
+  if (running_on_xbox_) {
+    return {bounds.Width + (bounds.X), bounds.Height + (bounds.Y)};
+  }
+
+  if (physical) {
+    // Return the height in physical pixels
+    return {bounds.Width * static_cast<float>(disp.RawPixelsPerViewPixel()),
+            bounds.Height * static_cast<float>(disp.RawPixelsPerViewPixel())};
+  }
+
+  return {bounds.Width, bounds.Height};
+}
+
+float FlutterWindowWinUWP::GetDpiScale(
+    winrt::Windows::Graphics::Display::DisplayInformation const& disp) {
+  float dpi = disp.LogicalDpi();
+  auto raw_dpi = disp.RawDpiX();
+  auto raw_width = disp.ScreenHeightInRawPixels();
+  auto scale = disp.ResolutionScale();
+  auto raw_per_view = disp.RawPixelsPerViewPixel();
```

Comment:
Nit: prefer actual types to `auto` here since the RHS doesn't make the type obvious.

Reply:
Weird, I thought i'd fixed these.. fixed in next commit.